integration-testing/src/main/java/io/grpc/testing/integration/TestServiceImpl.java

```diff
@@ -293,7 +294,14 @@ private void scheduleNextChunk() {
         Chunk nextChunk = chunks.peek();
         if (nextChunk != null) {
           scheduled = true;
-          executor.schedule(dispatchTask, nextChunk.delayMicroseconds, TimeUnit.MICROSECONDS);
+          if (executor instanceof ScheduledExecutorService) {
```

Comment:
Do you think there is any advantage to ever using ScheduledExecutor? The original intent was to add delay but I'm not sure we ever care.  Maybe just get rid of it entirely?